docs/skopeo-inspect.1.md

```diff
@@ -77,6 +77,10 @@ The username to access the registry.
 
 The password to access the registry.
 
+**--no-tags**, **-n**=_bool_
```

Comment:
Just noticed on the page that the options are not listed in alpha order.  While you're in there, could you rearrange them please?

Reply:
Hello @TomSweeneyRedHat, 

I had a look at some other man files too and they too don't have it in alphabetical order. Given that, do you still want this to be changed for this doc content in this PR? If yes, let me know and I'll go ahead and do it.